clock/clock.go

```diff
@@ -1,18 +1,57 @@
 package clock
 
 import (
-	"github.com/jonboulle/clockwork"
+	"time"
 )
 
 // Clock provides an interface that packages can use instead of directly
-// using the time module, so that chronology-related behavior can be tested.
+// using the time module, so that chronology-related behavior can be tested
+// Adapted from: https://github.com/jonboulle/clockwork
```

Comment:
@ianjdarrow Could you please confirm that this is all that's required here? I was expecting more, like a reference to the license under which we've copied and adapted this code, a copyright line, ...